src/plugins/install.js

```diff
@@ -144,10 +148,16 @@ module.exports = function (Plugins) {
 	};
 
 	Plugins.isActive = async function (id) {
+		if (nconf.get('plugins:active')) {
+			return nconf.get('plugins:active').includes(id);
+		}
 		return await db.isSortedSetMember('plugins:active', id);
 	};
 
 	Plugins.getActive = async function () {
+		if (nconf.get('plugins:active')) {
+			return nconf.get('plugins:active');
+		}
 		return await db.getSortedSetRange('plugins:active', 0, -1);
 	};
```

Comment:
Hmm, you've already updated this method. You should use this method in all the other places where `plugins:active` is checked directly.

Reply:
Done in most places. Ironically the only place it couldn't be used is in `plugins/data.js`, as importing this file there would have side effects (setting two additional pubsub listeners on primary instance)